docs/inspector-plot.ipynb

```diff
@@ -0,0 +1,77 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "c5913ed6-9577-4614-990b-2f80bce1ebf8",
+   "metadata": {},
+   "source": [
+    "# Inspector plot\n",
+    "\n",
+    "The `inspector` plot is for three-dimensional data.\n",
```

Comment:
I think "1-D plot with 1-D profile." as we had it in the old plotting, with adding vertical line markers was pretty useless.
For that, I am thinking more of having a `superplot` (like Lamp's superplot), which is like the `slicer` plot: you slice 2d data to get a 1d plot with a slider. In addition, you have buttons to save the current line. I think it's better than clicking on the axes to see a cut in the other dimension.

"2-D plot with inspected values displayed in a table.": good idea :+1:

Reply:
> I think "1-D plot with 1-D profile." as we had it in the old plotting, with adding vertical line markers was pretty useless.

Not necessarily: Consider `{'spectrum':1_000_000, 'tof':1000}`. We cannot make a 2-D plot anymore, but someone might want to make a 1-D plot of the counts per spectrum and then inspect and compare TOF-spectra from different pixels.

Reply:
What you suggest is a little different, because the `inspector` plot does no summing along the third dimension.
So in your example, the first plot wouldn't be the counts per spectrum, it would be the counts per spectrum in the first tof bin.

I guess you could easily make this by adding a summing node in the graph.

Reply:
> What you suggest is a little different, because the inspector plot does no summing along the third dimension.

Without sum this does not seem very useful? In many cases a single slice does not contain useful values to display?

Reply:
So are you saying that `inspector` should also sum along the third dimension?

Reply:
Should we allow different operations, such as `sum, mean, min, max` ?

Reply:
👍

Reply:
> So are you saying that `inspector` should also sum along the third dimension?

Yes.